data_service/adapters/storage/gcs.py

```diff
@@ -50,6 +55,7 @@ def __download_partitioned_file_from_storage(self, path: str) -> str:
 
         os.makedirs(blob_download_path)
 
+        # Is this path to partitioned parquet valid considering todays structure?
```

Comment:
I'm not sure this path is valid any more? the key for the bucket doesn't match the file system nearly at all, and does not contain a version number 🤔

Reply:
Do we need to focus on this issue now? We walk through all details when we move to cloud, anyway.

Reply:
We can let this module be until we migrate to cloud 👍🏻